updater/internal/path/path.go

```diff
@@ -0,0 +1,46 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package path
+
+import "path/filepath"
+
+const (
+	latestDirFragment         = "latest"
+	rollbackDirFragment       = "rollback"
+	serviceFileDirFragment    = "install"
+	serviceFileBackupFilename = "backup.service"
+)
+
+// LatestDirFromTempDir gets the path to the "latest" dir, where the new artifacts are,
+// from the temporary directory
+func LatestDirFromTempDir(tmpDir string) string {
```

Comment:
So, I have this task to rip out the tmpDir parameter, and when I do that I want to rewrite these functions to be more based on a single anchor point (installDir) instead of all relative.

Can we come back to this on that future PR, and discuss it then? I think it'll be less generic then.